tests/otel_integrations/test_openai.py

```diff
@@ -240,7 +240,7 @@ def request_handler(request: httpx.Request) -> httpx.Response:
     elif request.url == 'https://api.openai.com/v1/completions':
         json_body = json.loads(request.content)
         if json_body.get('stream'):
-            chunks = [
+            completion_chunks = [
```

Comment:
Is this change useful? if not please revert for a cleaner diff

Reply:
Prevents warnings due to incompatible types.